build-system/tasks/process-github-issues.js

```diff
@@ -0,0 +1,274 @@
+        }
+        else if (milestoneTitle === 'Prioritized FRs' ||
+          milestoneTitle === 'New FRs') {
+          updates.push(applyLabel(issue, 'Type: Feature Request'));
+        }
+        else if (milestoneTitle === 'Backlog Bugs' ||
+          milestoneTitle.startsWith('Sprint')) {
+          updates.push(applyLabel(issue, 'Type: Bug'));
+        }
+        // Apply default priority if no priority
+        if (hasPriority == false && milestoneTitle != 'New FRs' &&
+          milestoneTitle !== '3P Implementation' &&
+          milestoneTitle !== 'Pending Triage' && milestone != null) {
+          updates.push(applyLabel(issue, 'P2: Soon'));
+        }
+        return Promise.all(updates);
+      });
+    });
+    assert(Array.isArray(issues), `issues must be an array.`);
+    return issues;
+  });
+}
+
+/**
+ * @param {string} issue
+ * @param {number} milestoneNumber
+ * @return {!Promise<*>}
+ */
+function applyMilestone(issue, milestoneNumber) {
+  var options = extend({}, optionsMilestone);
+  options.qs = {
+    state: 'open',
+    per_page: 1000,
+    access_token: GITHUB_ACCESS_TOKEN,
+  };
+
+  issue.milestone = milestoneNumber;
+  return githubRequest(
+      '/issues/' + issue.number,
+      'PATCH',
+      issue.milestone, 'milestone').then(function() {
+        if (verbose) {
+          util.log(util.colors.green(
+              'Milestone applied ' + milestone + ' for #' + issue.number));
+        }
+      });
+}
+
+/**
+ * @param {string} issue
+ * @param {string} label
+ * @return {!Promise<*>}
+ */
+function applyLabel(issue, label) {
+  var options = extend({}, optionsMilestone);
+  options.qs = {
+    state: 'open',
+    per_page: 1000,
+    access_token: GITHUB_ACCESS_TOKEN,
+  };
+  return githubRequest(
+      '/issues/' + issue.number + '/labels',
+      'POST',
+      [label], 'label').then(function() {
+        if (verbose) {
+          util.log(util.colors.green(
+              'Label applied ' + label + ' for #' + issue.number));
+        }
+      });
+}
+
+/**
+ * @param {string} path
+ * @param {string=} opt_method
+ * @param {*} opt_data
+ * @param {string} typeRequest
+ * @return {!Promise<*>}
+ */
+function githubRequest(path, opt_method, opt_data, typeRequest) {
+  var options = {
+    url: 'https://api.github.com/repos/ampproject/amphtml' + path,
+    body: {},
+    headers: {
+      'User-Agent': 'amp-changelog-gulp-task',
+      'Accept': 'application/vnd.github.v3+json'
+    },
+    qs: {
+      access_token: GITHUB_ACCESS_TOKEN
+    },
+  };
+  if (opt_method) {
+    options.method = opt_method;
+  }
+  if (opt_data) {
+    options.json = true;
+    if (typeRequest === 'milestone') {
+      options.body['milestone'] = opt_data;
+    }
+    else options.body = opt_data;
+  }
+  return request(options)
+}
+
+gulp.task('process-github-issues', 'Get issues data', processIssues, {
+  options: {
+    dryrun: '  Generate process but dont push it out',
```

Comment:
nit: `don't` is missing an apostrophe. (You'll have to escape it here by using `don\'t`)